src/app/core/component.ts

```diff
@@ -265,31 +275,28 @@ export default class Component {
      *
      */
     getFeatureIDs() {
-        return this.__features;
+        return this._features;
     }
 
-    /**
-     * Checks if the component has default parameters
-     * @param {String} key Key to access the component
-     * @returns {boolean} Returns true whether it has default parameters or not
-     * @memberof Component
-     */
-    hasDefaultParam(key) {
-        if (this.getDefaults().hasOwnProperty(key)) return true;
-        else return false;
-    }
+    // /**
+    //  * Checks if the component has default parameters
+    //  * @param {String} key Key to access the component
+    //  * @returns {boolean} Returns true whether it has default parameters or not
+    //  * @memberof Component
+    //  */
+    // hasDefaultParam(key: string) {
+    //     if (this.getDefaults().hasOwnProperty(key)) return true;
+    //     else return false;
+    // }
```

Comment:
if its not getting called, delete it

Reply:
I didn't realize that it is actually being used